CHANGES.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## Dendrite 0.11.0 (2023-01-20)
+
+### Features
+
+* Added `/_dendrite/admin/purgeRoom/{roomID}` to clean up the database
+* The default room version was updated to 10 (contributed by [FSG-Cat](https://github.com/FSG-Cat))
+
+### Fixes
+
+* The last three missing server-server Sytests - bringing us to **100%** Synapse parity 🎉
```

Comment:
Move this to the top-level between ## Dendrite 0.11.0 and ### Features so it's more clear, else it's easy to miss. Also make it clear this is from a federation perspective, not CS API.